static/views/options.html

```diff
@@ -4,6 +4,7 @@
 <head>
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="icon" type="image/png" href="assets/icons/alby_icon_yellow_32x32.png">
```

Comment:
@escapedcat this path might be relevant for v3. - there was some change in the paths, wasn't it?

Reply:
It should not matter for the metatags but will check